### ISSUE TITLE
Is putting "fine tune dark areas" on gamebar for easy access possible?

### ISSUE THREAD

**Issue by geahad** [Maassoft/ColorControl#366]
I appologize if this was asked before, but is putting "fine tune dark areas" from expert --> other for LG OLED TVs possible? I don't see the option...

**Comment by Maassoft** [comment]
No, that setting can currently not be added to the GameBar. I will make it possible in the next release.
